jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/StartEventTest.java

```diff
@@ -424,9 +425,9 @@ public void afterProcessStarted(ProcessStartedEvent event) {
         });
 
         assertEquals(0, list.size());
-        for (int i = 0; i < 2; i++) {
-            Thread.sleep(1200);
-        }
+        // Timer in the process takes 1000ms, so after 2 seconds, there should be 2 process IDs in the list.
+        // 2100ms is here because it is important to be sure that 2 seconds have passed.
+        Thread.sleep(2100);
```

Comment:
I made some changes to this. Take a look if it is OK. The main difference is that I created a constant that defines extra time needed for Thread.sleep() calls so the sleep time should be easily configurable in the future. Now it is 250ms. I think that this should be enough now to cover all tests that are in StartEventTest class because now there are no loops with Thread.sleep().